### ISSUE TITLE
Provide an easy way of getting the lernraum address

### ISSUE THREAD

**Issue by Mimoja** [lernranz/lernranz#16]
A one-click-solution might be usefull?

**Comment by clonejo** [comment]
You mean a link to eg. Openstreetmap? Like http://www.openstreetmap.org/?mlat=50.77989&mlon=6.07620#map=19/50.77989/6.07620 ?

**Comment by Mimoja** [comment]
Yes and no. Even if OSM is the cooler project, nearly all smartphones have GoogleMaps. I think it would be pretty handy to choose a free room, tap on it and get navigation. What do you think?

**Comment by clonejo** [comment]
On my Android phone this works fine for both Google Maps and Osmand (OSM viewing and routing app):
`geo:50.7798,6.0738?z=16&q=50.7798,6.0738(Foo+bar)` (unfortunately, Github doesn't like this URI)

Can you test that on an iOS device?

Since desktop browsers do not support the geo uri, we can link to OSM there.
